autorag/retriever/google_and_vector_retriever.py

```diff
@@ -0,0 +1,95 @@
+import os
+from llama_index.schema import TextNode, NodeWithScore, QueryBundle
+from llama_index.indices.vector_store.retrievers.retriever import VectorIndexRetriever
+from llama_index.core.base_retriever import BaseRetriever
+import requests
+from bs4 import BeautifulSoup
+from googleapiclient.discovery import build
+from typing import Any, Dict, List, Optional
```

Comment:
`Any, Dict, Optional` are imported but not used. Suggest to remove to avoid any unused imports.

Reply:
good catch